internal/pkg/service/stream/storage/event/sender.go

```diff
@@ -17,7 +17,8 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
 )
 
-const componentID = keboola.ComponentID("keboola.keboola-stream")
+// Schema: https://github.com/keboola/event-schema/blob/main/schema/ext.keboola.keboola-buffer..json
+const componentID = keboola.ComponentID("keboola.keboola-buffer")
```

Comment:
We have new component stream, but we call it `buffer`. I don't like it. As we would like to deprecate it from project, this way we include telemetry compatibility. What's the use case of this PR? We would like to inspect new telemetry with `buffer` query or where we face problem of backward compatibility?

Reply:
https://keboolaglobal.slack.com/archives/C054DF6DZ1B/p1721213248279279?thread_ts=1721047861.443159&cid=C054DF6DZ1B